charts/prometheus-pushgateway/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "v1.7.0"
+appVersion: "v1.8.0"
 description: A Helm chart for prometheus pushgateway
 name: prometheus-pushgateway
-version: 2.8.0
+version: 2.9.0
```

Comment:
```suggestion
version: 2.10.0
```

Reply:
@zeritti but its bumped, from 2.8.0 to 2.9.0

Reply:
I see now, nvm